### PR TITLE
[Doc] update docs/deployment/sr_operator.md

### PR DESCRIPTION
**Pull request by yandongxiao**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/deployment/sr_operator.md [hunk 1]
```diff
@@ -151,12 +151,12 @@ From outside the Kubernetes cluster, you can access the StarRocks cluster throug
 
     ```YAML
     starRocksFeSpec:
-        image: starrocks/alpine-fe:2.4.1
-        replicas: 3
-        requests:
+      image: starrocks/fe-ubuntu:latest
```
Comment (kevincai) [inline docs/deployment/sr_operator.md:154]: if here uses `latest`, and then upgrade to `3.0-latest`, it is actually a downgrade giving current release.

**Review by kevincai — APPROVED** [review]

**Review by hellolilyliuyi — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by EsoragotoSpirit — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
@mergify backport branch-3.1 branch-3.0 branch-2.5

**Pull request merged**